src/lesson8/classwork/table_creating_views.sql

```diff
@@ -0,0 +1,67 @@
+CREATE VIEW products_suppliers_categories AS
```

Comment:
It would be better to write "CREATE OR REPLACE VIEW" insted of "CREATE VIEW". Please, check other views also.